docs/src/docs/asciidoc/index.adoc

```diff
@@ -906,6 +943,65 @@ To run it use the following:
 
 	./gradlew :samples:hazelcast-spring:tomcatRun
 
+[[api-jdbcoperationssessionrepository]]
+=== JdbcOperationsSessionRepository
+
+`JdbcOperationsSessionRepository` is a `SessionRepository` implementation that uses Spring's `JdbcOperations` to store sessions in a relational database.
+In a web environment, this is typically used in combination with `SessionRepositoryFilter`.
+Please note that this implementation does not support publishing of session events.
+
+[[api-jdbcoperationssessionrepository-new]]
+==== Instantiating a JdbcOperationsSessionRepository
+
+A typical example of how to create a new instance can be seen below:
+
+[source,java,indent=0]
+----
+include::{indexdoc-tests}[tags=new-jdbcoperationssessionrepository]
+----
+
+For additional information on how to create and configure a `JdbcTemplate`, refer to the Spring Framework Reference Documentation.
+
+[[api-jdbcoperationssessionrepository-config]]
+==== EnableJdbcHttpSession
+
+In a web environment, the simplest way to create a new `JdbcOperationsSessionRepository` is to use `@EnableJdbcHttpSession`.
+Complete example usage can be found in the <<samples>>
+You can use the following attributes to customize the configuration:
+
+* **tableName** - the name of database table used by Spring Session to store sessions
+* **maxInactiveIntervalInSeconds** - the amount of time before the session will expire in seconds
+
+===== Custom LobHandler
+
+You can customize the BLOB handling by creating a Bean named `springSessionLobHandler` that implements `LobHandler`.
+
+===== Custom ConversionService
+
+You can customize the default serialization and deserialization of the session by providing a `ConversionService` instance.
+When working in a typical Spring environment, the default `ConversionService` Bean (named `conversionService`) will be automatically picked up and used for serialization and deserialization.
+However, you can override the default `ConversionService` by providing a Bean named `springSessionConversionService`.
+
+[[api-jdbcoperationssessionrepository-storage]]
+==== Storage Details
+
+By default, this implementation uses `SPRING_SESSION` table to store sessions. Note that the table name can be easily customized as already described.
+Depending on your database, the table definition can be described as below:
+
+[source,sql]
+----
+CREATE TABLE SPRING_SESSION (
```

Comment:
It would probably be good to use an include pointing to the actual classpath resource files we use.